src/components/track/mobile/BottomButtons.tsx

```diff
@@ -29,37 +31,61 @@ const messages = {
 const BottomButtons = (props: BottomButtonsProps) => {
   const { toast } = useContext(ToastContext)
 
-  return (
+  const repostButton = (
```

Comment:
render fns for these too https://github.dev/AudiusProject/audius-client/blob/637c9b4f337c117ee54ecead959693e77990e825/src/containers/download-buttons/DownloadButtons.tsx#L89